xml/mysql_banners.xml

```diff
@@ -132,6 +138,8 @@
     <param pos="0" name="os.family" value="Linux"/>
     <param pos="0" name="os.product" value="Linux"/>
     <param pos="2" name="os.version"/>
+    <param pos="0" name="os.certainty" value="0.80"/>
```

Comment:
I _think_ the only certainty values for MySQL fingerprints were added by me, specifically for OS.